<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="OGLWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="selectNeighberhoodGroup">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Select neighberhood</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>6</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>26</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="selectSearchAlgorithm">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Select search algorithm</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="settingsWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="useHiddenWallsCbx">
           <property name="text">
            <string>Use hidden walls</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runAlgorithmBtn">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGenerate_random"/>
    <addaction name="actionLoad_space"/>
    <addaction name="actionSave_space"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_result"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGenerate_random">
   <property name="text">
    <string>Generate random space</string>
   </property>
  </action>
  <action name="actionLoad_space">
   <property name="text">
    <string>Load space</string>
   </property>
  </action>
  <action name="actionSave_space">
   <property name="text">
    <string>Save space</string>
   </property>
  </action>
  <action name="actionOpen_result">
   <property name="text">
    <string>Open result</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">oglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
